docs/semantic-search.ipynb

```diff
@@ -55,333 +56,236 @@
-              "    </tr>\n",
-              "    <tr>\n",
-              "      <th>240003</th>\n",
-              "      <td>516000</td>\n",
-              "      <td>[-0.120020054, 0.024080949, 0.10693012, -0.018...</td>\n",
-              "      <td>{'text': ' Are the new Samsung Galaxy J7 and J...</td>\n",
-              "    </tr>\n",
-              "    <tr>\n",
-              "      <th>240004</th>\n",
-              "      <td>516001</td>\n",
-              "      <td>[-0.095293395, -0.048446465, -0.017618902, -0....</td>\n",
-              "      <td>{'text': ' I just watched an add for Indonesia...</td>\n",
-              "    </tr>\n",
-              "  </tbody>\n",
-              "</table>\n",
-              "</div>"
-            ],
-            "text/plain": [
-              "            id                                             values  \\\n",
-              "240000  515997  [-0.00531694, 0.06937869, -0.0092854, 0.003286...   \n",
-              "240001  515998  [-0.09243751, 0.065432355, -0.06946959, 0.0669...   \n",
-              "240002  515999  [-0.021924071, 0.032280188, -0.020190848, 0.07...   \n",
-              "240003  516000  [-0.120020054, 0.024080949, 0.10693012, -0.018...   \n",
-              "240004  516001  [-0.095293395, -0.048446465, -0.017618902, -0....   \n",
-              "\n",
-              "                                                 metadata  \n",
-              "240000  {'text': ' Why is a \"law of sciences\" importan...  \n",
-              "240001  {'text': ' Is it possible to format a BitLocke...  \n",
-              "240002  {'text': ' Can formatting a hard drive stress ...  \n",
-              "240003  {'text': ' Are the new Samsung Galaxy J7 and J...  \n",
-              "240004  {'text': ' I just watched an add for Indonesia...  "
-            ]
-          },
-          "execution_count": 2,
-          "metadata": {},
-          "output_type": "execute_result"
         }
       ],
       "source": [
-        "from pinecone_datasets import load_dataset\n",
-        "\n",
-        "dataset = load_dataset('quora_all-MiniLM-L6-bm25')\n",
+        "import os\n",
+        "from getpass import getpass\n",
         "\n",
-        "# The metadata we need is actually stored in the \"blob\" column so let's rename it\n",
-        "dataset.documents.drop(['metadata'], axis=1, inplace=True)\n",
-        "dataset.documents.rename(columns={'blob': 'metadata'}, inplace=True)\n",
+        "def get_pinecone_api_key():\n",
+        "    \"\"\"\n",
+        "    Get Pinecone API key from environment variable or prompt user for input.\n",
+        "    Returns the API key as a string.\n",
         "\n",
-        "# We don't need sparse_values for this demo either so let's drop those as well\n",
-        "dataset.documents.drop(['sparse_values'], axis=1, inplace=True)\n",
+        "    Only necessary for this example. When using Pinecone yourself, \n",
```

Comment:
Maybe instead of `for this example` say `for notebooks`? It read a little weird to me and it's not just this example, it's all notebooks, right?

Reply:
Should I say "example notebooks", then? Basically I don't want people thinking the source tag is required.

Reply:
fixed thank you! I mis read what you meant here!